qa/python_models/bls_model_loading/config.pbtxt

```diff
@@ -37,7 +37,7 @@ output [
 
 instance_group [
   {
-    count: 3
+    count: 1
```

Comment:
Is this impacting any other tests?

Reply:
This is not part of my changes. I have no idea why it's here. Seems like some rebase went wrong.

Reply:
Changes gone with rebase 👍